src/app/(dashboard)/meetings/[meetingId]/page.tsx

```
@@ -1,3 +1,44 @@
+	);
+
+	return (
+		<HydrationBoundary state={dehydrate(queryClient)}>
+			<Suspense fallback={<MeetingIdViewLoading />}>
+				<ErrorBoundary fallback={<MeetingIdViewError />}>
+					<MeetingIdView meetingId={meetingId} />
+				</ErrorBoundary>
+			</Suspense>
+		</HydrationBoundary>
+	);
 	return <div>Meeting</div>;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)